include/robot_fingers/n_joint_blmc_robot_driver.hpp

```diff
@@ -242,10 +246,10 @@ class NJointBlmcRobotDriver
      * positive direction until the next encoder index.  The position of this
      * encoder index is the "home position".
      *
-     * By default, the zero position is the same as the home position.  The
-     * optional argument home_offset_rad provides a means to move the zero
-     * position
-     * relative to the home position.  The zero position is computed as
+     * By default, the zero position after homing is the same as the home
+     * position.  The optional argument home_offset_rad provides a means to move
+     * the zero position relative to the home position.  The zero position is
+     * computed as
      *
      *     zero position = encoder index position + home offset
```

Comment:
Since searching for the next index position for homing is only one of the possible homing modes now:

zero position = home position + home offset

Reply:
Thanks for pointing that out.  I updated the comment and while doing this, I noticed that it's a bit inconsistent to pass some arguments to the method and read others directly from `config_`, so I changed it to read everything from `config_` instead.